.github/workflows/pull_request.yml

```diff
@@ -16,6 +16,8 @@ name: ci
 
 on:
   pull_request:
+    paths-ignore:
+      - 'charts/**'
```

Comment:
A quick reference for the reviewer(re `paths-ignore` ) in case it is required : 
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths